twitchio/ext/eventsub/models.py

```diff
@@ -150,6 +231,40 @@ def verify(self):
         return web.Response(status=200, text=self.challenge)
 
 
+class ReconnectEvent(BaseEvent):
+    """
+    A reconnect event. Called by twitch when the websocket needs to be disconnected for maintenance or other reasons
+
+    .. note::
+        These are only dispatched when using :class:`~twitchio.ext.eventsub.EventSubWSClient
+
+    Attributes
+    -----------
+    reconnect_url: :class:`str`
+        The URL to reconnect to
+    connected_at: :class:`~datetime.datetime`
```

Comment:
```suggestion
    connected_at: :class:`datetime.datetime`
```
To be more inline with the rest of the PR.